cypress/integration/retention.js

```diff
@@ -11,6 +11,9 @@ describe('Retention', () => {
         // stable. This is to try and avoid an issue where after selecting a
         // filter property, the results section would be blank
         cy.get('[data-attr=retention-table').should('exist')
+        // tests for new property group filters
+        // cy.get('[data-attr=insight-retention-add-filter-group]').click()
+        // cy.get('[data-attr=property-select-toggle-0]').click()
```

Comment:
not needed anymore?

Reply:
it'll be needed when we remove the feature flags 😁

Reply:
Ah, true. Let's put a TODO there so it's noted

Reply:
ok!!